remote/gitea/parse.go

```diff
@@ -0,0 +1,107 @@
+package gitea
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/drone/drone/model"
+)
+
+const (
+	hookEvent       = "X-Gogs-Event"
```

Comment:
I think we can improve this with another PR. This PR just support commit status and create the new Gitea driver.